monkey/monkey_island/cc/services/initialize.py

```diff
@@ -11,12 +11,12 @@
 
 def initialize_services(data_dir: Path) -> DIContainer:
     container = DIContainer()
+    container.register_instance(AWSInstance, AWSInstance())
```

Comment:
This doesn't cause a 2 second delay?

Reply:
No, the AWSInstance constructor starts the requests on a thread. If you attempt to access its members before they're ready, those calls will block.